python/fledge/common/service_record.py

```diff
@@ -45,9 +45,9 @@ class InvalidServiceStatus(Exception):
         # TODO: tell allowed service status?
         pass
 
-    __slots__ = ['_id', '_name', '_type', '_protocol', '_address', '_port', '_management_port', '_status', '_token']
+    __slots__ = ['_id', '_name', '_type', '_protocol', '_address', '_port', '_management_port', '_status', '_bearer_token']
```

Comment:
I would suggest not to put _bearer_token in slots. keep a separate data structure (say svcRegistryTokens) to hold serviceName/ID: issuedBearerToken (private to service registry). if single use token match then add entry to svcRegistryTokens and use that to verify at each request for the services requires auth. (via a middleware) pass/no_pass

Reply:
this shall also mitigate the risk/chances of exposure/snooping of bearer tokens in service API and to other service.